CryptoPrices/Sources/Data/Sources/NetworkExtension/CoinRoute.swift

```diff
@@ -43,5 +45,12 @@ extension CoinRoute: Route {
         }
     }
 
-    public var parameterEncoding: ParameterEncoding? { .json }
+    public var parameterEncoding: ParameterEncoding? {
+        switch self {
+        case .myCoins: return .url
+            // TODO: Update when implement
```

Comment:
To keep the existing logic, can you help add a case `.trendingCoins` with `ParameterEncoding` as `.json` instead of removing it? thanks 🙏

Reply:
Oops, sorry 🙏 It's my bad when rebasing! Will update it now!

Reply:
@Thieurom just a friendly reminder that don't forget about fixing this issue 😁